fellow-submissions/mark-griffith/style.css

```diff
@@ -0,0 +1,51 @@
+body{
+    margin: 0;
+    font-size: 1.3rem;
+    font-family:Arial, Helvetica, sans-serif;
+    background-color:#a58686;
+}
+main{
+    display:grid;
+    grid-template-rows:1fr auto;
+    grid-template-columns:10% 3fr 3fr 3fr 10%;
+     grid-template-areas: 
+    ". header header header ."
+    ". aside section section ."
+    ". footer footer footer .";
+}
+section{
+    display:grid;
```

Comment:
This line is turning your `<section>` into a grid container as well. This not what you want. I think what you mean to say here is "section, you are an item in my grid". Well, this is implied. All the direct descendants of your grid container (`<main>`) know that they are grid items.